android/src/main/java/com/epicshaggy/biometric/NativeBiometric.java

```diff
@@ -128,7 +123,7 @@ public void isAvailable(PluginCall call) {
         JSObject ret = new JSObject();
 
         biometricManager = BiometricManager.from(getContext());
-        int canAuthenticateResult = biometricManager.canAuthenticate();
+        int canAuthenticateResult = biometricManager.canAuthenticate(BIOMETRIC_STRONG);
```

Comment:
Can we wrap this in if else?
```suggestion
        int canAuthenticateResult;
        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.P) {
          canAuthenticateResult = biometricManager.canAuthenticate(BIOMETRIC_STRONG);
        } else {
          canAuthenticateResult = biometricManager.canAuthenticate();
        }
```